arangod/Sharding/ShardingStrategy.cpp

```diff
@@ -36,6 +37,6 @@ bool ShardingStrategy::isCompatible(ShardingStrategy const* other) const {
 void ShardingStrategy::toVelocyPack(VPackBuilder& result) const {
   // only need to print sharding strategy if we are in a cluster
   if (ServerState::instance()->isRunningInCluster()) {
-    result.add("shardingStrategy", VPackValue(name()));
+    result.add(StaticStrings::ShardingStrategy, VPackValue(name()));
```

Comment:
this could have been a delete-only pr :disappointed:

Reply:
@maierlars : still a net deletion of 177 LOC :+1: